### PR TITLE
docs: Fix image links for buttons

### PR DESCRIPTION
**Pull request by Cristiandis**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,8 +23,8 @@ Builds can be found in the [Releases](https://github.com/revenge-mod/RevengeTwea
 
 ### Jailed
```
Comment (oSumAtrIX) [inline README.md:25]: I am not sure but the urls do not work (they are blank)

Comment (Cristiandis) [reply README.md:25]: they open for me

Comment (oSumAtrIX) [reply README.md:25]: https://github.com/user-attachments/assets/dd252453-8be2-45e7-a9fb-04fdeac7390f

Comment (Cristiandis) [reply README.md:25]: because they are not http links, they only work on iphones

Comment (Cristiandis) [reply README.md:25]: let me give you a vid

Comment (oSumAtrIX) [reply README.md:25]: Why is it a tinyurl link in the first place? This should be changed to a regular one. Images can also be uploaded directly in GitHub.

Comment (Cristiandis) [reply README.md:25]: https://files.catbox.moe/nvdmai.mov
file too big for github

Comment (oSumAtrIX) [reply README.md:25]: You can use https://streamable.com/ in the future but I get it.

**Review by oSumAtrIX — APPROVED** [review]

**Pull request merged**